.travis.yml

```diff
@@ -36,16 +28,16 @@ before_install:
     - echo session.gc_probability = 0 >> $INI
     - echo apc.enable_cli = 1 >> $INI
     - phpenv config-rm xdebug.ini || echo "xdebug not available"
-    - if [[ $TRAVIS_PHP_VERSION = 5.* ]]; then echo yes | pecl install -f apcu-4.0.11; fi
-    - if [[ $TRAVIS_PHP_VERSION = 7.* ]]; then echo yes | pecl install -f apcu_bc-1.0.4; fi
-    - if [[ $TRAVIS_PHP_VERSION = 7.* ]]; then echo yes | pecl install -f apcu-5.1.17; fi
+    - yes | pecl install -f apcu_bc-1.0.5
+    - yes | pecl install -f apcu-5.1.19
     - yes | pecl install uuid || true
     - php -i
 
 install:
     - export COMPOSER_ROOT_VERSION=dev-main
     - composer --prefer-source install
+    - if [[ $SYMFONY_PHPUNIT_VERSION = 9.* ]]; then export SYMFONY_PHPUNIT_CONFIG=phpunit-9.xml.dist; else export SYMFONY_PHPUNIT_CONFIG=phpunit.xml.dist; fi;
     - if [[ $TRAVIS_PHP_VERSION = nightly ]]; then phpenv global 7.1; ./vendor/bin/simple-phpunit install; phpenv global nightly; fi
 
 script:
-    - ./vendor/bin/simple-phpunit
+    - ./vendor/bin/simple-phpunit  -c $SYMFONY_PHPUNIT_CONFIG
```

Comment:
I'm including a different config file on PHPUnit 9 to disable the `TestListener`. See my comment here: https://github.com/symfony/symfony/issues/33331#issuecomment-713703230